source/_integrations/harmony.markdown

```diff
@@ -95,6 +95,14 @@ Start an activity. Will start the default `activity` from configuration.yaml if
 | `entity_id`            |       no | Entity ID to target.
 | `activity`             |      yes | Activity ID or Activity Name to start.
 
+### Service `remote.toggle`
+
+..........
```

Comment:
The description is missing, as pointed out by @klaasnicolaas already.

Please add one, thanks! 👍

Reply:
Yes, but I suppose that the service firstly should start working correctly.
Maybe there should be an Issue reported? Sorry but I am very new here. And even with this supplementing content suggestion, I am not sure that I am in the right place. My intention was to draw someone's attention that I found service working not accordingly to my logic expectation and with missing documentation. I was asked to do that, it wasn't my idea;)

Reply:
Oh, I'm sorry, I've missed a thing as well.

You are editing the Harmoney page, which is remote. All entities in Home Assistant inherit the services from the parent entity. So a Harmony gets all services from Harmony.

However, we only document services to the places they belong to. So the service is `remote.toggle`, which is part of the remote integration, not the Harmony. It is usable due to Harmony being an remote, but it should not be documented on the Harmony integration.

The reasoning for this is: If we start doing that, we are duplicating a tremendous amount of documentation everywhere, which would be hard to keep in sync.

So, Thanks for opening up a PR, but I'm going to close this one.


If you think there is a bug with the service, please report is @ <https://github.com/home-assistant/home-assistant/issues>

Thanks! 👍

Reply:
I can understand. But in this case on this page shouldn't be also: `remote.turn_on`, `remote.turn_off`, `remote.send_command`. 
But in this case I suggest making better descriptions of this services on page [/integrations/remote/](https://www.home-assistant.io/integrations/remote/)

Indeed I think there is a bug in `remote.toggle` but until I can not find anywhere description how designer intended it to work, it is only my guess/logic conception.